package.json

```diff
@@ -16,7 +16,8 @@
     "lint": "lerna run lint",
     "release:chat-ui": "lerna run release --scope='mongodb-chatbot-ui' && git add ./package-lock.json && git commit --amend --no-edit",
     "scripts:analyzeMessages": "lerna run analyzeMessages --scope='scripts'",
-    "scripts:scrubMessages": "lerna run scrubMessages --scope='scripts'"
+    "scripts:scrubMessages": "lerna run scrubMessages --scope='scripts'",
+    "server:start": "lerna run start --scope='chatbot-server-mongodb-public'"
```

Comment:
nit: I'd format this like `start:server` e.g. how we do `npm run dev:server` and `npm run dev:ui` as subsets of `npm run dev`

Reply:
fair. consistency would be good. scripts is the prefix on the above commands. will keep same here for now and make new ticket to update all

Reply:
https://jira.mongodb.org/browse/EAI-176

Reply:
To be clear I don't think we should switch the `scripts:<foo>` ones